pkgs/servers/misc/navidrome/default.nix

```diff
@@ -1,48 +1,48 @@
-{ lib, stdenv, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
+{ lib, stdenv, buildGoModule, fetchFromGitHub, ffmpeg, ffmpegSupport ? true,
+  makeWrapper, pkg-config, taglib, taglib_extras, zlib, callPackage, nixosTests}:
 
 with lib;
 
-stdenv.mkDerivation rec {
+buildGoModule rec {
   pname = "navidrome";
-  version = "0.47.5";
+  version = import ./version.nix;
 
+  # Can't use -trimpath because tests require accurate working dir.
+  allowGoReference = true;
```

Comment:
This leaves `go` as part of the runtime closure, which adds an extra ~500MB.

Reply:
Is there a way to set GOFLAGS to not include `-trimpath` so that `doCheck` can be left enabled?